instructure/canvas-lms
# Fix symlinks in canvas_quizzes client app

Fixes #713 

Test plan:
- ensure the compile assets succeeds and does not throw errors related to canvas_quizzes
- ensure that the Quiz Statistics app continues to work normally

A CLA is on file under my employer, Simon Fraser University.
